### DISCUSSION TITLE
Table 3 - Practical 4

### DISCUSSION THREAD

**Discussion by EleonoraMilea** [IBAHCM/RPiR#459] [Old]
Eleonora Milea: EleonoraMilea https://github.com/IBAHCM/EleonoraMileaBCI.git

**Comment by IsobelAJones** [comment]
Isobel Jones: https://github.com/IBAHCM/IsobelAJonesBCI.git

**Comment by lynseythomson** [comment]
my repo 
https://github.com/IBAHCM/lynseythomsonBCI.git 

:)

**Comment by EleonoraMilea** [comment]
Hey that's my repository :)
EleonoraMilea: https://github.com/IBAHCM/EleonoraMileaBCI.git

**Reply by EleonoraMilea** [reply]
Hey @NinaGangi @lynseythomson @IsobelAJones I have just completed 4.2.  Could you please check if my package is installing properly? Thank you :)

**Reply by EleonoraMilea** [reply]
I have also finished 4.3.. so can you please raise an issue in my repository? Thanks @NinaGangi @IsobelAJones @lynseythomson

**Comment by NinaGangi** [comment]
NinaGangi: https://github.com/IBAHCM/NinaGangiBCI.git
